libs/utils/utils.js

```diff
@@ -793,7 +793,13 @@ export async function loadIms() {
       },
       onError: reject,
     };
-    loadScript(`${base}/deps/imslib.min.js`);
+    const params = new URLSearchParams(window.location.search);
```

Comment:
I see query params are also fetcher [here](https://github.com/adobecom/milo/blob/6c13dded0e8dc1a7cd22504c684b6cad3477a473/libs/utils/utils.js#L833). Could we get them early on and then re-use wherever neccessary?